### ISSUE TITLE
logging metrics as bar charts

### ISSUE THREAD

**Issue by AlekseySh** [OML-Team/open-metric-learning#108]

**Comment by Rei-Moriaty** [comment]
Hi @AlekseySh 

This is my first time contributing to open source. I wanted to work on this. Can you give me a brief on what needs to be done?

**Comment by AlekseySh** [comment]
@Rei-Moriaty Yes, sure. If you run one of the examples (for instance, training model on INSHOP dataset) you will see a lot of graphs with metrics by different categories on Neptune.
The problem now is that it's hard to find the best/worst category based on these graphs. So, if we can log them as barplot images, it would be easier to navigate, where the x-axis is the category and the y-axis is the metric. And we need a series of these images with respect to epoch number.

PS. Probably Neptune provides a way of working with bar plots, so we can avoid using images.

Some random image from the internet: 
<img width="622" alt="Screenshot 2022-10-17 at 20 28 27" src="https://user-images.githubusercontent.com/20143575/196254545-51410e75-e9e1-4d97-864b-800b4e6a5e21.png">

**Comment by isehmi** [comment]
@AlekseySh im happy to work on this if its still an active problem

but before I started was playing around on colab and ran into a problem with adding `from oml.lightning.entrypoints.train import pl_train` I get `ModuleNotFoundError` 

I checked out #146  and was able to run visualization.ipynb  just fine am i missing a step for `oml.lightning.entrypoints.train` or is that no accessible via `pip install git+`

**Comment by AlekseySh** [comment]
@isehmi Hi, it would be fine, because it's still an active problem 

As for imports, ideally, all of the have to be accessible in collab env 
1) Do other imports work in collab?
2) Can you do this import locally without collab?

**Comment by isehmi** [comment]
@AlekseySh sorry for taking so long to get back to you 

when i install locally i have the same problem when i just use `pip install -U open-metric-learning` but i found that installing pytorch and pytorch lightning works. if this is a step that the users needs to do i recommend its added somewhere in the installation guide. 

i still can't get it to work on google colab even with installing the extra modules 

note this problem only exist with `lightning ` modules from what i can tell everything else works without installing those modules

**Comment by AlekseySh** [comment]
@isehmi Hm, after what you said, I created an empty docker container with python and successfully installed the library from PIP. Which OS do you have?

**Comment by isehmi** [comment]
Windows 10

**Comment by AlekseySh** [comment]
I opened the dedicated issue regarding this: https://github.com/OML-Team/open-metric-learning/issues/222

**Comment by AlekseySh** [comment]
@isehmi But I guess, it's not a blocker for you If you want to work on this issue. Because you will install the repo via `git clone` from your fork, not from the PIP index. I hope, in this setup, you will be able to contribute. But yep, we need to figure out what is wrong with Windows.

**Comment by isehmi** [comment]
@AlekseySh as I'm still working on windows I found another small bug that effects windows users should i be making issue for these or will this all be done when testing on windows systems starts

**Comment by AlekseySh** [comment]
@isehmi I think after we start testing on windows we find more, so, the creating one particular issue is not needed now

**Comment by isehmi** [comment]
thats what i was thinking a bit pointless adding issue one at a time. when the time comes we will need to build a robust system to catch all possible issue. i will still keep track of any problems i encounter as it might come in handy

**Comment by AlekseySh** [comment]
@isehmi that's right! thank you

**Comment by AlekseySh** [comment]
We are still looking for someone who can take care of this issue :)

**Comment by nqthinh04** [comment]
@AlekseySh Hello, I would like to retackle this issue. I want to ask if there has been any updates on the task, and if your requirements/preferences has changed since your initial posting.

Thanks for reading, hope to hear from you soon.
